doc/syntax/operators.rdoc

```diff
@@ -0,0 +1,22 @@
+= Operators
+
+In Ruby, operators such as +++, are defined as methods on the class.
+Literals[link:/literals.rdoc] define their methods within the lower level, C language.
+{String Class}[rdoc-ref:string.c], for example.
```

Comment:
Source code files will not be included generated files.
Bare class name is linked to the page for that class, instead.

```suggestion
String class, for example.
```

Reply:
Good to know! I'll remove the link refernce.